docs/book/v3/storage/adapter-plugin-manager.md

```diff
@@ -0,0 +1,28 @@
+# Adapter Plugin Manager
+
+The `AdapterPluginManager` extends the laminas-servicemanager `AbstractPluginManager`, and has the following behaviors:
+
+- It will only return `Laminas\Cache\Storage\StorageInterface` instances.
+- All services are not shared by default; a new instance will be created each time you call `get()`.
+
+## Factory
+
+`Laminas\Cache\Storage\AdapterPluginManager` is mapped to the factory `Laminas\Cache\Service\StorageAdapterPluginManagerFactory` when wired to the dependency injection container.
+
+The factory will be automatically registered when loading/installing the `Laminas\Cache` module in `laminas-mvc` and/or loading/installing the `ConfigProvider` into a Mezzio application.
+
+Since version 3.11.0, the factory will look for the `config` service, and use the `storage_adapters` configuration key to seed it with additional services.
```

Comment:
The version hints must be highlighted:

## Version Hints

If a new feature was added with a minor release then a version hint must be added to the related section of the documentation page.
The hint must be added after the headline of the new feature.

An admonition must be used for a version hint.

### Syntax Example

``````
INFO: **New Feature**
Available since version 3.11.0
``````

Reply:
Can this be added to the very first line of this markdown or where do I have to put that?